### PR TITLE
Managers intial data scripts

### PR DESCRIPTION
**Pull request by tristinwells**

### CODE REVIEW

**Comment by tristinwells** [comment]
@troygrosfield this is what I came up with.

**Comment by troygrosfield** [comment]
Write a test for this that proves it does what you say it does. 
- [ ] move the "management" folder inside your site's main app folder "personalsite
- [ ] create "tests" folder inside "personalsite" folder
- [ ] add file in "tests" directory called "test_management_command.py" that will be the test file for you management commands
- [ ] create a test for your management command that proves running the command does indeed create new blog posts for the user you provided.

File: management /commands/datascript.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    def superuser(self):
+        obj, created = Person.objects.get_or_create(superuser='tristin-admin')
+
+    def non_superuser(self):
+        obj, created = Person.objects.get_or_create(
+            superuser='tristin',
+            defaults={'blog 1': date(2015, 5, 20),
+                      'blog 2': date(2015, 5, 30),
+                      'blog 3': date(2015, 5, 25)})
```
Comment (troygrosfield) [inline management /commands/datascript.py]: This all needs to be handled in two separate cases.  Keep this method's purpose to just create the "Person" object.  Then create a new method that accepts a user parameter and creates three blog entries for that user.  Something like:

``` python
def non_superuser(self):
    # this method should only contain fields about the user.  So your defaults should 
    # be a dict of user fields only.  The key is the field, the value is the field value.
    return Person.objects.get_or_create(...)

def create_blog_entries(self, user):
    """The user is the user to create blog entries for."""
    # create 3 blog entries for the user here Blog.objects.create(...)
```

File: templates/index.html [hunk 2]
```diff
@@ -64,10 +64,15 @@ <h3 class="text-muted">Meet Tristin Wells</h3>
         
           <h4><a href="/about">About</a></h4>
           <p>Come see what I am all about!</p>
-          <p><iframe width ="100%" height = "300" scrolling="Yes" 
+          
+          <p><iframe width="100%" height="300" scrolling="Yes" 
           src="https://moving.media/movies/5/paradise/embed"></iframe>
           
           </p>
+          <iframe width = "100%" height = "300" scrolling = "Yes"
```
Comment (troygrosfield) [inline templates/index.html:72]: remove the spaces between the keys and the values.

**Pull request merged**